public/app/percona/environment-overview/components/ContactWidget/Contact.tsx

```diff
@@ -0,0 +1,76 @@
+import { AppEvents } from '@grafana/data';
+import { Icon, IconButton, IconName, useStyles2 } from '@grafana/ui';
+import { logger } from '@percona/platform-core';
+import appEvents from 'app/core/app_events';
+import { getPerconaUser } from 'app/percona/shared/core/selectors';
+import { isApiCancelError } from 'app/percona/shared/helpers/api';
+import React, { useCallback, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useCancelToken } from '../../../shared/components/hooks/cancelToken.hook';
+import { WidgetWrapper } from '../WidgetWrapper/WidgetWrapper';
+import { CONTACT_CANCEL_TOKEN } from './Contact.constants';
+import { Messages } from './Contact.messages';
+import { ContactService } from './Contact.service';
+import { getStyles } from './Contact.styles';
+import { CustomerSuccess } from './Contact.types';
+
+const Contact = () => {
+  const [pendingRequest, setPendingRequest] = useState(true);
+  const [data, setData] = useState<CustomerSuccess>();
+  const { isPlatformUser } = useSelector(getPerconaUser);
+  const [generateToken] = useCancelToken();
+  const styles = useStyles2(getStyles);
+
+  const getData = useCallback(async (showLoading = false) => {
+    showLoading && setPendingRequest(true);
+    try {
+      const contact = await ContactService.getContact(generateToken(CONTACT_CANCEL_TOKEN));
+      setData(contact);
+    } catch (e) {
+      if (isApiCancelError(e)) {
+        return;
+      }
+      logger.error(e);
+    }
+    setPendingRequest(false);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    if (isPlatformUser === true) {
+      getData(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isPlatformUser]);
+
+  const copyToClipboard = useCallback(async () => {
```

Comment:
We already have a component that has this functionality, `ClipboardButton`. Please check `ChunkedLogsViewer.tsx`

Reply:
Can you specify path to it? My editor can't find it.